routes/user.js

```diff
@@ -96,12 +98,50 @@ const deleteUser = (request, response) => {
         });
 };
 
+const recipesByPantry = (request, response) => {
```

Comment:
The content of this method is good, but I think it fits better within `CurrentPantryServices`. My recommendation would be to consider moving this into that file and update it a bit-

```
const recipesByUserPantry = (request, response) => {
  const { userId, } = request.params;
  const pantryItems = await CurrentPantryServices.getPantryItemsOfUser(userId);
  let baesAPIURL.......
  .....
  // rest of the code stays the same.

```